app/src/main/res/layout/dialog_buy_app_info_header.xml

```diff
@@ -5,16 +5,16 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:layout_marginEnd="15dp"
     android:layout_marginStart="15dp"
     android:layout_marginTop="14dp"
+    android:layout_marginEnd="15dp"
     tools:showIn="@layout/fragment_express_checkout_buy"
     >
   <ImageView
       android:id="@+id/app_icon"
       android:layout_width="42dp"
       android:layout_height="42dp"
-      android:src="@drawable/ic_appc"
+      tools:src="@drawable/ic_appc"
```

Comment:
This should not be changed. In case the application is not installed and we are not able to find its icon.